templates/pod_pruner.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -exuo pipefail
+OC='oc -n {{ ocp_namespace }}'
+
+delete_threshold=$(date --date="{{ osbs_pruner_pods_days_old }} days ago" --iso-8601=seconds
+--utc)
+
+old_pods=$(
+    $OC get pods -o json | \
+    jq '.items[].metadata | select(.creationTimestamp < "'${delete_threshold}'") | .name' -r | \
```

Comment:
Does string comparison of ISO dates work as expected? It probably should, just thinking out loud :smile: 

The output from `date` seems to show timezone a bit differently, but it shouldn't be a problem
```shell
$ echo null | jq '"2021-10-07T00:00:00Z" < "2021-10-07T00:00:00+00:00"'
false

$ echo null | jq '"2021-10-07T00:00:00Z" < "2021-10-07T00:00:01+00:00"'
true
```

Reply:
I copied pruner script from old OCP 3 project. I am not even sure if it will be ever used. Clean up ownership should be defined in ticket CLOUDBLD-7635. In the meantime I would leave it like this and wait for resolution. :)